libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -418,6 +418,19 @@ void AP_Camera::send_camera_settings(mavlink_channel_t chan)
     }
 }
 
+// send camera capture status message to GCS
+void AP_Camera::send_camera_capture_status(mavlink_channel_t chan)
+{
+    WITH_SEMAPHORE(_rsem);
+
+    // call each instance
+    for (uint8_t instance = 0; instance < AP_CAMERA_MAX_INSTANCES; instance++) {
+        if (_backends[instance] != nullptr) {
+            _backends[instance]->send_camera_capture_status(chan);
```

Comment:
I've copied the way the other messages (e.g. `CAMERA_SETTINGS`, `CAMERA_INFORMATION`) are handled. 

However, these handlers are problematic if there are multiple camera backends:
* these messages do not have IDs (MAVLink intent is to use `compid` to distinguish components;
* however, all of these are sent with the `MAV_COMP_ID_AUTOPILOT1` `compid`, which will confuse a GCS.

Reply:
Raised #24955.